Source/Tools/FEXLoader/LinuxSyscalls/Syscalls/Thread.cpp

```diff
@@ -248,6 +256,12 @@ namespace FEX::HLE {
         Thread->ThreadManager.clear_child_tid = child_tid;
       }
 
+      if (IsVFork) {
+        // Close the read end of the pipe.
+        // Keep the write end open so the parent can poll it.
+        close(VForkFDs[0]);
```

Comment:
Can we move the initial `close`s of both the child and the parent to the `fork`? It's a bit randomly placed here, and moving it closer to the fork makes it easier to quickly see which of the FDs is still valid in each branch of execution.

Reply:
Moved.